userspace/libsinsp/event.cpp

```diff
@@ -138,7 +138,7 @@ sinsp_threadinfo* sinsp_evt::get_thread_info(bool query_os_if_not_found)
 		return m_tinfo;
 	}
 
-	return &*m_inspector->get_thread_ref(m_pevt->tid, query_os_if_not_found, false);
+	return m_inspector->get_thread_ref(m_pevt->tid, query_os_if_not_found, false).get();
```

Comment:
:+1: to all these, though at some point we should probably switch to smart pointers